MyNotes/Form1.cs

```diff
@@ -68,31 +74,31 @@ private void buttonCreate_Click(object sender, EventArgs e)
         /// <param name="e"></param>
         private void buttonSave_Click(object sender, EventArgs e)
         {
-            if (File.Exists(fileDirectory.Text)) //если файл сущетсвует
-            {
-                File.AppendAllText(fileDirectory.Text, textBoxNote.Text); //записываем в него текст
```

Comment:
if you use AppendAllText, and click save multiple times, you will end up with a file containing multiple copies of your text.

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.io.file.appendalltext?view=net-6.0